src/utils/semanticUtils.js

```diff
@@ -15,4 +15,5 @@ export const colors = [
 ]
 export const sizes = ['mini', 'tiny', 'small', 'medium', 'large', 'big', 'huge', 'massive']
 export const textAlignments = ['left', 'center', 'right', 'justified']
+export const vertAlignments = ['top', 'middle', 'bottom']
```

Comment:
For consistency and predictability please avoid abbreviations.  Let's spell this out entirely, `verticalAlignements`.